src/main/java/com/jfrog/ide/idea/scan/ScanManagersFactory.java

```diff
@@ -163,7 +162,9 @@ private ScanLogic createScanLogic(Utils.ScanLogicType type, ScanCache scanCache,
     }
 
     private void createScanManagers(Map<Integer, ScanManager> scanManagers, Set<Path> scanPaths, ExecutorService executor) throws IOException {
```

Comment:
I think we better add a description for this method. Specifically describe what the ```scanPaths``` set includes.